client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -560,16 +620,7 @@ export default function CompositeCheckout( {
 		} )
 	) {
 		debug( 'rendering empty cart page' );
-		const goToPlans = () => {
-			recordEvent( {
-				type: 'EMPTY_CART_CTA_CLICKED',
```

Comment:
Since this is being removed, we should also remove it from client/my-sites/checkout/composite-checkout/record-analytics.js

Reply:
The tracks related issues should be resolved, I also tested locally to ensure the track is still firing correctly after I click 'Go back'

 
![image](https://user-images.githubusercontent.com/16580129/142697746-18047433-451c-4f7d-9518-24084dc60cb6.png)